pilot/pkg/config/clusterregistry/clusterregistry.gotmpl

```diff
@@ -0,0 +1,16 @@
+apiVersion: clusterregistry.k8s.io/v1alpha1
+kind: {{.Kind}}
+metadata:
+  name: {{.Name}}
+  annotations:
+    config.istio.io/pilotEndpoint: "{{.PilotIP}}:9080"
```

Comment:
make the port configurable as well. 9080 is for old v1 stuff.

Reply:
this file is ONLY used in unit testing, so no any exposure to any inside or outside service.